src/logic/operate.js

```diff
@@ -2,24 +2,33 @@
 const Big = require('big.js');
 
 const operate = (numOne, numTwo, operation) => {
-  const first = Big(numOne);
-  const second = Big(numTwo);
+  const first = numOne ? Big(numOne) : null;
+  const second = numTwo ? Big(numTwo) : null;
+
 
   if (operation === '-') {
     return first.minus(second).toString();
-  } if (operation === '+') {
+  }
+
+  if (operation === '+') {
     return first.plus(second).toString();
-  } if (operation === 'x') {
+  }
+
+  if (operation === 'x') {
     return first.times(second).toString();
-  } if (operation === '÷') {
+  }
+
+  if (operation === '÷') {
     if (second === 0) {
```

Comment:
I think it's safer to test if `numTwo` is zero. 

Currently, you haven't handle division by zero properly.

![caldiverr0](https://user-images.githubusercontent.com/36057474/73165272-cab03e80-40f3-11ea-8b86-0c3798b01b9e.gif)